frontend/packages/knative-plugin/src/components/functions/FunctionsPage.tsx

```diff
@@ -44,15 +44,28 @@ const FunctionsListPage: React.FC<React.ComponentProps<typeof ListPage>> = (prop
         selector={{ matchLabels: { 'function.knative.dev': 'true' } }}
       />
     </KnativeServiceTypeContext.Provider>
+  );
+};
+
+const FunctionsListPage: React.FC<React.ComponentProps<typeof ListPage>> = (props) => {
+  const { t } = useTranslation();
+  const { ns } = useParams();
+  const [perspective] = useActivePerspective();
+  return perspective === 'dev' ? (
```

Comment:
I think we should look at removing this behavior from the dev side. I never really understood it, and it's inconsistent with admin. Why force users to pick a project if they have permission to see the resource across all projects? It's artificially limiting what they can do. Why remove that feature? I don't see what's complicated or confusing about it.

Not for this PR, but I think it's worth asking @kevinhatchoua and @itsptk about. I don't know how large an effort it is to change. If it's just removing the `CreateProjectListPage` from the various list pages, it might be worth making a run at as part of dev / admin convergence. I'd argue improving consistency is part of that effort.

@kevinhatchoua @itsptk This is the behavior where the dev perspective forces you to pick a project before showing you the list instead of just showing you all resources in the cluster when "All Projects" is selected.

Reply:
It was added as target users for developer perspective are non-admin and they do not have access to all projects.